### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by karachevtsev**
### HTML-CSS-Popup

[Demo](https://karachevtsev.github.io/html-css-popup/)
[Code base](https://github.com/karachevtsev/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Review by Temu4 — CHANGES_REQUESTED** [review]
Really cool approach for this popup! Thanks for that! 👍 
But there a couple things that you can improve)

File: submissions/karachevtsev/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          <div class="nav-details__popup">
+            <div class="popup">
+              <div class="popup__container">
+                <div class="popup__apps apps">
+                  <ul class="apps__list">
+                    <li class="apps__item app-item">
+                      <a href="#"><img src="images/calendar.png" alt="" width="40" height="40">Calendar</a>
+                    </li>
+                    <li class="apps__item app-item">
+                      <a href="#"><img src="images/mail.png" alt="" width="40" height="40">Mail</a>
```
[the remark this answers is not in the document]
Comment (Temu4) [reply submissions/karachevtsev/html-css-popup/index.html:34]: No needed for decorative images [link](https://developers.google.com/web/fundamentals/accessibility/semantics-builtin/text-alternatives-for-images)

[the remark this answers is not in the document]
Comment (karachevtsev) [reply submissions/karachevtsev/html-css-popup/index.html]: Added alt attribute for all images

File: submissions/karachevtsev/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                    </li>
+                    <li class="apps__item app-item">
+                      <a href="#"><img src="images/news.png" alt="" width="40" height="40">News</a>
```
[the remark this answers is not in the document]
Comment (karachevtsev) [reply submissions/karachevtsev/html-css-popup/index.html]: Set new icons images. Removed dimensions attributes from img tag and set dimension in css

File: submissions/karachevtsev/html-css-popup/scss/_nav-details.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    &:hover {
+      background-color: lighten($theme-color, 50%)
+    }
+  }
+
+  &__popup {
+    position: absolute;
+    top: 140%;
+    right: -40px;
```
[the remark this answers is not in the document]
Comment (karachevtsev) [reply submissions/karachevtsev/html-css-popup/scss/_nav-details.scss]: Added media queries and updated mobile style

[the remark this answers is not in the document]
Comment (Temu4) [reply submissions/karachevtsev/html-css-popup/scss/_nav-details.scss]: +

File: submissions/karachevtsev/html-css-popup/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (Temu4) [reply submissions/karachevtsev/html-css-popup/index.html]: Hi @karachevtsev!
The core of this issue isn't related with images sources, but with their real sizes.
Please try to decrease them. You can use such free and simple tools as Paint, for example.
Screenshot 2020-12-22 at 11 25 45

**Comment by A-Ostrovnyy** [comment]
@Temu4 please finish review this PR)

**Comment by Temu4** [comment]
> Hi @karachevtsev!
> The core of this issue isn't related with images sources, but with their real sizes.
> Please try to decrease them. You can use such free and simple tools as Paint, for example.
> Screenshot 2020-12-22 at 11 25 45

Hey @karachevtsev !
Do you have some issues regarding that?
Please feel free to ping me in any messengers.

**Comment by karachevtsev** [comment]
Hi @Temu4 !
Resolve isuue with icons sizes.

**Comment by Temu4** [comment]
Good job! 👍

**Pull request merged**